### PR TITLE
LPS-178732 LPS-176550 cause about -3.18% performance regression

### PR DESCRIPTION
**Pull request by W2jx**
Hi @dantewang , I have fixed it and tested it locally.
Hi @zhangyan0701 Could you please help me verify if feature flag setting works on the cluster after the test passes ? Thank you.

Previous: https://github.com/liferay-core-infra/liferay-portal/pull/4718

### CODE REVIEW

File: modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/manager/FeatureFlagPreferencesManager.java [hunk 1]
```diff
@@ -68,8 +110,33 @@ private PortalPreferences _getPortalPreferences(long companyId) {
...
+			methodHandler, true);
+
+		clusterRequest.setFireAndForget(true);
+
+		_clusterExecutor.execute(clusterRequest);
+	}
+
 	private static final String _NAMESPACE = FeatureFlagConstants.FEATURE_FLAG;
 
+	private static final MethodKey _onNotifyMethodKey = new MethodKey(
```
Comment (dantewang) [inline modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/manager/FeatureFlagPreferencesManager.java:131]: @W2jx Thanks a lot! Could you also change this field name to match the actual method? Thanks!

**Comment by W2jx** [comment]
Hi @dantewang , I changed the name of the filed to `_methodKey`, do I need to change it to something else?

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:sf

**Comment by dantewang** [comment]
@W2jx Better to be "[methodName]MethodKey".

**Comment by Kvale1733** [comment]
ci:test:stable

**Comment by Kvale1733** [comment]
@W2jx No regressions found.

I tested the scenarios Summer outlined here: https://github.com/liferay-core-infra/liferay-portal/pull/4718#issuecomment-1587512464

Scenario 1 now passes. 

Scenario 2 still fails but it doesn't looks specific to this pr. I tested on master without cluster and even there you can still make a supplier account after disabling the flag.

I tested with COMMERCE-8087 (Enables Import/Export Center in Applications panel) and that worked fine. Disabling this flag on node two also disabled it on node 1. And the Import/Export center could not be accessed after disabling it on node 2.

So the feature flag settings works fine with clustering on this pr.

**Comment by W2jx** [comment]
only do rename, no need to rerun test

**Comment by W2jx** [comment]
Hi @dantewang , I have renamed it.

**Comment by zhangyan0701** [comment]
FYI. The automation tests will be added in https://liferay.atlassian.net/browse/LRQA-81076. 
cc @vicnate5

**Comment by dantewang** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/4815

**Pull request closed without merging**